docs/components/connectivity/wifi.md

```diff
@@ -1,4 +1,103 @@
 # WiFi
 
-!!! warning
-    This page is still in progress. You can check the description of blocks for help.
+## Overview
+
+The **WiFi** component can be used to manage the WiFi settings as well as to retrieve information about device's WiFi.
+
+## Methods
+
+### BSSID
+![](/assets/images/components/connectivity/wifi/m_bssid.png)
+
+\[None\]
+
+Returns the basic service set identifier (BSSID) of the current access point.
+
+### Disable
+![](/assets/images/components/connectivity/wifi/m_disable.png)
+
+\[ None \]  
+
+Disable the Wi-Fi
+
+### Enable
+![](/assets/images/components/connectivity/wifi/m_enable.png)
+
+\[ None \]
+
+Enable the Wi-Fi
+
+### Is 5GHz Supported
+![](/assets/images/components/connectivity/wifi/m_is-5ghz-supported.png)
+
+\[ None \]
+
+Return whether this adapter supports 5GHz band.
+
+* `True` - 5GHz band is supported by the adapter.
+* `False` - 5GHz band is not supported by the adapter.
+
+### Is Enabled
+![](/assets/images/components/connectivity/wifi/m_is-enabled.png)
+
+\[ None \]
+
+Return whether Wi-Fi is enabled or disabled.
+
+* `True` - WiFi is enabled.
```

Comment:
True/False methods should follow this pattern:
https://docs.kodular.io/components/utilities/color-utilities/#is-dark-color